rootfs-builder/rootfs.sh

```diff
@@ -38,18 +38,27 @@ typeset -r CONFIG_ARCH_SH="config_${ARCH}.sh"
 # build_rootfs() function.
 typeset -r LIB_SH="rootfs_lib.sh"
 
+# rootfs distro name specified by the user
+typeset distro=
+
+# path to the init executable / symlink
```

Comment:
Can you define "path" - is this host path or rootfs path? Is the path allowed to be relative or must it be absolute?

Reply:
I'll add clarifications, thanks